client/src/pages/UserProfile.jsx

```diff
@@ -4,31 +4,65 @@ import UserTable from '../components/presentational/profile/UserTable';
 import UserEvents from '../components/presentational/profile/UserEvents';
 import UserTeams from '../components/presentational/profile/UserTeams';
 import { UserProvider, UserContext } from '../context/userContext';
+import { Box, Typography, Grid } from '@mui/material';
 
 const UserProfile = (props) => (
   <UserProvider>
-    <div>
-      <div className="profile__header">
-        <h3 className="profile__title">My Profile</h3>
-      </div>
+    <Box>
+      <Box style={{ backgroundColor: '#bad3ff' }}>
+        <Typography
```

Comment:
Curious if one of these is redundant, I tought that Typography variant="h3" would make this component an h3 element already